docs/plugins.md

```diff
@@ -77,6 +77,15 @@ Snowpack will automatically call this function to load your plugin. That functio
   1. (optional) user-provided config options (`pluginOptions`)
 
 
+### Develop and Test
+
+To develop and test plugin, the strategy is the same as with other npm packages
+
+1. use `npm link` to make global the plugin with the future npm name `<plugin-name>`, and 
+2. use `npm link <plugin-name>` in the project you want to test it and
+3. in the project's `snowpack.config.json` under the plugins name provide the same `<plugin-name>` and optional plugin options
```

Comment:
```suggestion
<ol>
  <li>run <code>npm link<code> in your plugin’s project folder.</li>
  <li>create a new, example Snowpack project in a different location for testing</li>
  <li>in your example Snowpack project, run <code>npm link my-snowpack-plugin && npm install</code> (use the name from your plugin’s <code>package.json</code>)</li>
  <li>
    in your example Snowpack project, add your plugin to the <code>snowpack.config.json</code> along with any plugin options you’d like to test:
    <pre>
      <code class="json">
{
  "plugins": [
    ["my-snowpack-plugin", { "option-1": "testing", "another-option": false }]
  "
}
      </code>
    </pre>
  </li>
</ol>
```

I love this section! Adding this is really helpful.

But I think we could add just a few more details to make it clear. You don’t have to keep all these suggestions, but I’d suggest improving #3 on the list. I have a hard time imagining the instructions “in the project's `snowpack.config.json` under the plugins name.” I’d suggest including a JSON snippet for easier following (tip: you can use HTML to place a code block in an ordered list)?

Reply:
Thanks, accepted everything with some additional tweaks, and also changed the order:

`npm install && npm link my-snowpack-plugin` as `npm install` would, sadly and silly, clean `npm link` symlinks otherwise

Reply:
> `npm install && npm link my-snowpack-plugin` as` npm install` would, sadly and silly, clean npm link symlinks otherwise

Ah I always forget! You can tell how good I am with linking packages from memory 😄